src/aspire/source/image.py

```diff
@@ -27,7 +29,50 @@
 logger = logging.getLogger(__name__)
 
 
-class ImageSource:
+class ImageAccessor:
```

Comment:
I'm a little wary of pulling this into the public API. One option is to leave it here in the third level (and not import it into `aspire.source.__init__`). Another is to make it private (so `_ImageAccessor`). Latter may be preferable.

Reply:
Ended up doing both. We can just import from the third level in the one other place it's used (`aspire/source/simulation.py`) so I'm not sure if the `_` is necessary? I'm fine with either